src/main/java/io/jenkins/plugins/pipelinegraphview/consoleview/PipelineConsoleViewUrlProvider.java

```diff
@@ -0,0 +1,32 @@
+package io.jenkins.plugins.pipelinegraphview.consoleview;
+
+import hudson.Extension;
+import hudson.model.Descriptor;
+import hudson.model.Run;
+import jenkins.console.ConsoleUrlProvider;
+import org.jenkinsci.Symbol;
+import org.jenkinsci.plugins.workflow.job.WorkflowRun;
+import org.kohsuke.stapler.DataBoundConstructor;
+
+public class PipelineConsoleViewUrlProvider implements ConsoleUrlProvider {
+
+    @DataBoundConstructor
+    public PipelineConsoleViewUrlProvider() { }
+
+    @Override
+    public String getConsoleUrl(Run<?, ?> run) {
+        if (run instanceof WorkflowRun) {
+            return run.getUrl() + PipelineConsoleViewAction.URL_NAME;
+        }
```

Comment:
From a very quick look, I noticed that the visualization in this plugin does not show the overall log in any form, so for example, if your Pipeline has no stages, it shows has a blank page, and things like messages around Jenkins restarting will not be shown, which could be problematic in practice, but at least for now this is just a PoC for the new API.

Reply:
for now https://github.com/jenkinsci/pipeline-graph-view-plugin/pull/289/commits/5e9e5a2fd7d9ece86375eb4badae19dae1b98656 should be enough I think

<img width="462" alt="image" src="https://github.com/jenkinsci/pipeline-graph-view-plugin/assets/21194782/b9e9f748-588e-4f36-98ea-48f3cc41541c">